### ISSUE TITLE
Possible design flaw with BTT manunfacturing process. 

### ISSUE THREAD

**Issue by ph0tic** [bigtreetech/Eddy#78]
(luke@bigtree-tech.com pls read:)

(this is for the BTT EDDY **USB** version): got a request in to BTT on amazon for replacement so... fingers crossed there.  But I thought i would try to let someone - **_anyone_** - know a possible reason why this thing is so inconsistent in its success rate, configuration issues, and so on:

I've been having the countless issues with my BTT Eddy others have been having as well. the cant read probe, eddy probe error, probe already triggered, all that stuff - **I made macros to go through the steps over and over again:** `BTT_EDDY_STEP1`, `BTT_EDDY_STEP2` `BTT_EDDY_STEP3` ...

My reg_drive_current after a lot of testing became 16 _and even consistently 17_ (as in, the calibrate was actually _recommending_ it) -- **_But not Everyon has a 10x AmScope to actually look at these things..._**

(Disclaimer, I'm a hobbyist. I didn't know what a ceramic capacitor was until 3 days ago):

1. Taking it apart I noticed some interesting stuff. First, there is a "conflict of angles between the probe PCB and inner slot/guides inside the case.
![image](https://github.com/user-attachments/assets/a812dceb-54a1-4304-a7e8-6dd3cc7f2e8c)
(I didnt mark this in the image BUT inside the blue circle you can see a piece of plastic that acts as a "guide" to force the sensor PCB down.) 

2. When putting the case back together, there was a gap between the pcb and the shell; when under pressure the shell forces the gab to close, putting even more pressure on that connection with the  solder connection to the sensor:
![image](https://github.com/user-attachments/assets/c5efaa3e-d5c9-4b8e-a5c0-4245ee088eaf)

So what do these two things mean? The BTT Eddy is putting some stress in a place that there SHOULD NOT BE. Solder may be metal and have some flex, but what it connects to MIGHT NOT. Which brings me to the next points.

3. under a 10x microscope, I actually found a minor gap between the pad and and the PCB it was supposed to be part of. It is difficult to see but putting pressure on that part showed some flex; the nub on the top left of the solder is actually the extension of the pad that is supposed to make contact with the Ceramic Capacitor AFAIK.  It may look close to contact with the PCB , but i promise you it was MUCH FARTHER from the board when i pushed down on it to see if my eyes were deceiving me:
![image](https://github.com/user-attachments/assets/8e8a4eae-7f64-4880-957d-6f6919d1ff6e)

4. On the left (bottom in image) of the capacitor, and both sides of thermistor, Solder aws applied not only to the pads on the PCB but directly contacting the SMD. on the **RIGHT** (top in image) solder was _**NOT**_ directly contacting the SMD. 
![image](https://github.com/user-attachments/assets/c375eca1-6fb2-4fa4-be0f-92ff19b0f637)

I Did some micro-soldering and extended the solder on the right side of the ceramic capacitor (bypassing the loose pad). I also may have kinda-sorta... maybe... lost the thermistor on the right entirely - _**BUT**_ using the MCU sensor as my temperature probe.... 

 _**It now works BETTER than when i purchased it. **_ (still want a new one though, miss that probe thermistor). It is actually reliable like i have heard so many people say before. 
 
 Futher investigation showed what this youtuber had done, https://youtu.be/0sucEbgGC_0?si=M3mSOVA57AwH3pl5&t=716, and i knew i was on the right track. he Redid the whole thing - and i think he did prior to even using it once - and said it worked perfect.
 
So thats it! you have a flaw in your production/manufacturing. Those weak pads on the sensor board most likely _**CANNOT**_ take the stress of whatever is happening during completion (putting together) of the probes. So _**you end up with some that work perfectly out the gate, and some that have no exact fix**_ 

tldr; if you have a lot of a product that work in completely different ways, chances are its not the people using it. :( 

The concept of the Eddy probe is amazing. Yours is very compact and not a huge horizontal thing. Its a good design. But these flaws are kill you. 

Somehow i want to keep using this thing - even through it 
- destroyed two Z-endstops (due to very sketchy probe_calibate auto script) 
- ruined my Fulament PEI sheet by grinding, and 
- made a 0.4mm into a 0.7mm nozzle)

**Comment by acwest** [comment]
I had an issue on my Eddy Duo where the solder joints on the USB port were cold-soldered, and the port popped off when I put the USB connector in. I have experience with microelectronics, so was able to resolder the connecter, including using much beefier solder connections on the side channels of the connector to give it more mechanical strength
